karate-core/src/main/java/com/intuit/karate/Runner.java

```diff
@@ -179,6 +179,9 @@ public List<FeatureCall> resolveAll() {
                 if (ko.paths != null) {
                     paths = ko.paths;
                 }
+                if (ko.threads > 0) {
```

Comment:
I think it is 1 by default because of the code in `Main.java` - so this may ALWAYS be true and not honor what was set via the builde . maybe `ko.threads != 1` is better logic, can you test this ?

Reply:
Should be make it ko.threads > 1 or !=1 both seems fine to me, The reason why I have avoided ko.threads!=1 is what if value of threads set via build is by mistake negative or something like zero.  Though this condition is also handled below, in code at https://github.com/karatelabs/karate/blob/v1.4.0/karate-core/src/main/java/com/intuit/karate/Runner.java#L271.

Reply:
@vipin1326 we could have a case where builder set it to 2 in the code but we want to force it to 1 via the env and `karate.options`, does that make sense ?

Reply:
@ptrthomas got your point, I have changed the check to ko.threads != 1. Also tested the same.